controller/helper/process-survey-instances.js

```diff
@@ -22,15 +20,36 @@ function processSurveyInstances (surveys) {
 /**
  * Takes in a Survey Instances and get the % time left to be shown on complience chart
  * @param {Object} surveys - list of survey instances
+ * @returns {Object} processed list of datetimes
+ */
+function pickDates (surveys) {
+    const dates = surveys.map((survey) => {
+        return moment(new Date(survey.startTime).toISOString()).utc().format('MM/DD/YYYY HH:mm');
+    });
+
+    const lastDate = moment(new Date(surveys[0]
+        .dateCompleted)
+        .toISOString()).utc().format('MM/DD/YYYY HH:mm');
+    const missingValue = -1;
+
+    if (dates.indexOf(lastDate) <= missingValue) {
```

Comment:
how I'm reading the logic here:

1. you get the first date, name it `lastDate`
2. check if the `lastDate` exists in the same list it came from
3. the `lastDate` does not exist in the list, add it

that doesn't really make sense to me.
What is the intention here?